packages/tasks/src/tasks/text-to-image/inference.ts

```diff
@@ -44,17 +48,9 @@ export interface TextToImageParameters {
 	 */
 	seed?: number;
 	/**
-	 * The size in pixel of the output image
+	 * The width in pixels of the output image
 	 */
-	target_size?: TargetSize;
-	[property: string]: unknown;
-}
-/**
- * The size in pixel of the output image
- */
-export interface TargetSize {
-	height: number;
-	width: number;
+	width?: number;
```

Comment:
Is it possible to keep `target_size` with `@deprecated` ?

```suggestion
	width?: number;
	
	/** 
	 * @deprecated use {@link width} and {@link height} instead
	 */
	 target_size?: TargetSize;
```

To not break existing stuff and not do another major version bump

cc @SBrandeis

Reply:
yes, I don't mind keeping it and just deprecate it instead (done in [23888c7](https://github.com/huggingface/huggingface.js/pull/1158/commits/23888c7601fdf43989753d45341f5a387c55040c))

Reply:
(if it never worked we can probably just remove it though no?)

Reply:
yes if it never worked we can remove

Reply:
I double checked again, it never worked, so I removed it completely (same on hfh side).